testsuite/src/it/java/org/graalvm/tests/integration/AppReproducersTest.java

```diff
@@ -818,6 +817,68 @@ public void monitorFieldOffset(TestInfo testInfo, Apps app) throws IOException,
         }
     }
 
+    @Test
+    @Tag("builder-image")
+    @IfMandrelVersion(min = "23.1.5", inContainer = true)
+    public void forSerializationContainerTest(TestInfo testInfo) throws IOException, InterruptedException {
+        forSerialization(testInfo, Apps.FOR_SERIALIZATION_BUILDER_IMAGE);
+    }
+
+    @Test
+    @IfMandrelVersion(minJDK = "23.1.5")
```

Comment:
This should be:

```suggestion
    @IfMandrelVersion(min = "23.1.5")
```

no?